crates/ruff_linter/src/rules/flake8_bandit/rules/hardcoded_sql_expression.rs

```diff
@@ -4,15 +4,22 @@ use ruff_python_ast::{self as ast, Expr, Operator};
 
 use ruff_diagnostics::{Diagnostic, Violation};
 use ruff_macros::{derive_message_formats, violation};
-use ruff_python_ast::str::raw_contents;
-use ruff_source_file::Locator;
 use ruff_text_size::Ranged;
 
 use crate::checkers::ast::Checker;
 
+// Implementation of regex from bandit
 static SQL_REGEX: Lazy<Regex> = Lazy::new(|| {
-    Regex::new(r"(?i)\b(select\s.+\sfrom\s|delete\s+from\s|(insert|replace)\s.+\svalues\s|update\s.+\sset\s)")
-        .unwrap()
+    Regex::new(
+        r"(?ixs)
```

Comment:
What's the motivation for including additional modifiers like `x` and `s`? We don't use any comments in the regex so `x` shouldn't be required. And the `\s` group should include the newline character so we might not require the `s` modifier as well. I might be wrong here so it would be helpful if you can provide some context here :)

Reply:
I wanted to implement the exact RegEx available in bandit's codebase, so translated `re.DOTALL` to `?(s)`.
I couldn't recollect the reason why did I add `x` modifier though. Removed x and ran tests, which passes all tests successfully.

Reply:
I would try to keep the changes to our pattern minimal to avoid unwanted changes to the rule's semantic